tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/CiamIntegrationTests.cs

```diff
@@ -168,6 +168,14 @@ public async Task OBOCiam_CustomDomain_ReturnsValidTokens()
             string ciamWebApi = "634de702-3173-4a71-b336-a4fab786a479";
             string ciamEmail = "idlab@msidlabciam6.onmicrosoft.com";
 
+            //Get lab details
+            var labResponse = await LabUserHelper.GetLabUserDataAsync(new UserQuery()
+            {
+                FederationProvider = FederationProvider.CIAMCUD,
+                SignInAudience = SignInAudience.AzureAdMyOrg,
+                PublicClient = PublicClient.no
```

Comment:
Why would this not be publicClient?

Reply:
Because the way the App has been setup and exposed

Reply:
in the list shared to lab app, this public app was not there. Added now